build.ps1

```diff
@@ -823,6 +826,11 @@ function Build-Foundation($Arch, [switch]$Test = $false)
       $Targets = @("default", "install")
     }
 
+    # Turn off safeseh for lld as it has safeseh enabled by default
+    # and fails with an ICU data object file icudt69l_dat.obj. This
+    # matters to X86 only.
+    $ExtraSwiftArgs = "-Xlinker /SAFESEH:NO"
```

Comment:
We should conditionalise it to x86 and pass it as `CMAKE_Swift_FLAGS`.

Reply:
Done